modules/osint/username_search.py

```diff
@@ -58,15 +59,18 @@ def check(self, url, site, data):
 			OUTPUT['links'][site] = url
 
 def module_api(self):
-	query = self.options['query']
-	filepath = os.path.join(os.getcwd(), 'data', 'username_checker.json')
-	with open(filepath) as file:
-		data = json.loads(file.read())
-	thread(self, data, query,self.options['thread'])
-	output = OUTPUT
+        query = self.options['query']
+        project_root = up(up(up(__file__)))
+        filepath = os.path.join(project_root, 
+                                'data', 
+                                'username_checker.json')
+        with open(filepath) as file:
+                data = json.loads(file.read())
```

Comment:
I think you can replace with `data = json.load(file)`

Reply:
will do